gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -150,7 +170,7 @@ TFLiteCore::setInputTensorProp ()
   auto input_idx_list = interpreter->inputs ();
   inputTensorMeta.num_tensors = input_idx_list.size ();
 
-  for (int i = 0; i < inputTensorMeta.num_tensors; i++) {
+  for (int i = 0; i < inputTensorMeta.num_tensors; ++i) {
```

Comment:
BTW, what is difference between i++ and ++i  in this code?

Reply:
According to the below answer, it looks almost same.
https://stackoverflow.com/a/24887
I just tried to make it better from any type of performance issue.